### PR TITLE
feat: add demo

### PR DESCRIPTION
**Pull request by NingLu**
*Issue #, if available:*
Test

*Description of changes:*
Test

*How Has This Been Tested:*
Test

*[x] My testing has passed*

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by yanbasic** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: nodejs-build-cloudfront-extensions
* Commit ID: dbbb7ffa759d207e03b0c142cd2e4ff4df55e997
* Result: FAILED
* [Build Logs](https://uc0zdtvc9j.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=29c546a9-631c-41c3-bc78-4041bacac517%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by yanbasic** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: python-build-cloudfront-extensions
* Commit ID: 40c434d852c79a7ca949626c1ecb87e496fd7b6f
* Result: FAILED
* [Build Logs](https://uc0zdtvc9j.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=5056f6c1-3ea1-42a6-92bd-c89eee2bf3b4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by yanbasic** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: nodejs-build-cloudfront-extensions
* Commit ID: 40c434d852c79a7ca949626c1ecb87e496fd7b6f
* Result: FAILED
* [Build Logs](https://uc0zdtvc9j.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=59ada8db-19fd-4992-84f0-d72434524a06%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: edge/python/ab3-demo/prewarm/app.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+def lambda_handler(event, context):
+    url_list = get_url_from_s3()
+    pop_split = pop_list.split(',')
+
+    mapping_json = json.loads(cf_mapping)
+
+    for pop in pop_split:
+        pop = pop.strip()
+        for url in url_list:
+            if len(url) > 0:
```
Comment (NingLu) [inline edge/python/ab3-demo/prewarm/app.py:61]: Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`.

 [Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)

Similar issue at line number 69.

**Pull request closed without merging**